pom.xml

```diff
@@ -420,9 +420,14 @@
         <version>${commons-math3.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.openjdk.nashorn</groupId>
-        <artifactId>nashorn-core</artifactId>
-        <version>${nashorn.version}</version>
+        <groupId>org.graalvm.js</groupId>
+        <artifactId>js-scriptengine</artifactId>
+        <version>${graalvm.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.graalvm.js</groupId>
+        <artifactId>js</artifactId>
+        <version>${graalvm.version}</version>
```

Comment:
Both libraries are licensed as UPL. It's fine as it's classified as [Category A](https://www.apache.org/legal/resolved.html).
- https://mvnrepository.com/artifact/org.graalvm.js/js-scriptengine/23.0.8
- https://mvnrepository.com/artifact/org.graalvm.js/js/23.0.8